unit_tests/test_aggregator.py

```diff
@@ -404,12 +366,9 @@ def test_delete_aggregator(aggregator_explicit):
 
 
 def test_add_to_batch_commands(aggregator_explicit):
-    # pylint: disable=protected-access
     buffer_instance = aggregator_explicit.add_to_batch_commands
     assert buffer_instance == aggregator_explicit._client_command_buffer
 
 
 def test_commands_buffer_length(aggregator_explicit):
-    patch("gsy_e_sdk.aggregator.ClientCommandBuffer.buffer_length",
```

Comment:
The exact same patch is performed at the top of the module

Reply:
Similar situation as in this comment [here](https://github.com/gridsingularity/gsy-e-sdk/pull/195/files/697cab28a6b68be7ace854535033cb16c47a4be6#r764905505).